docs/index.rst

```diff
@@ -107,23 +117,19 @@ community projects, contributions, suggestions, fixes and constructive feedback.
    contribute.rst
 
 Further reading
-===============
+---------------
 
 You can read this guide offline in different formats, if you install one of
 the `binary packages <BinPkgs_>`_.
```

Comment:
We will probably want to update the source of this link, since this still points to Launchpad. Although the Launchpad version will still be there - Dave or Dominik can probably confirm which is the best to use.

Reply:
@waveform80 is currently working on this, but I think the link will not change. The binary package is just outdated.